lib/components/Editor/CustomExtensions/Markdown/index.js

```diff
@@ -36,6 +44,15 @@ const MarkdownEditor = ({
 
   const handleChange = (e) => {
     setContent(e.target.value);
+    onChange(e.target.value);
+  };
+
+  // Submit on Ctrl+Enter and Cmd+Enter
+  const handleSubmit = (e) => {
+    if (e.keyCode === 13 && (e.metaKey || e.ctrlKey)) {
```

Comment:
can we move this 13 to a constant like 'KEYBOARD_KEY_CODE_ENTER' or something like that?